pkg/users/account-create-dialog.js

```diff
@@ -199,10 +230,13 @@
         if (field == "locked")
             state.change_passw_force = false;
 
+        if (field == "uid")
+            state.uid_exists = false;
```

Comment:
This added line is not executed by any test.  [Details](https://cockpit-logs.us-east-1.linodeobjects.com/pull-18353-20230215-154634-5d29fa30-fedora-37-devel//Coverage/lcov/github-pr.diff.gcov.html)